autofish.lua

```diff
@@ -0,0 +1,220 @@
+-- handles automatic fishing jobs to limit the number of fish the fortress keeps on hand
+-- autofish [enable | disable] <max> [min] [--include-raw | -r]
+
+--@ enable=true
```

Comment:
You'll also need `--@ module=true` here too so the state change hooks can be attached
docs link (though I know you've already read this):
https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#enabling-and-disabling-scripts